### PR TITLE
Product service

### PR DESCRIPTION
**Pull request by Hegabovic**
- added product services 
- added product model 
- **solved a conflict in index.php file**

### CODE REVIEW

File: Model/DatabaseConnector.php [hunk 1]
```diff
@@ -0,0 +1,32 @@
+<?php
+require_once("vendor/autoload.php");
+
+use Illuminate\Database\Capsule\Manager as DBC;
+
+class DatabaseConnector
+{
+    private $dbc;
+
+    public function __construct()
```
Comment (MohamedEldefrawy) [inline Model/DatabaseConnector.php:10]: connection string List should be replaced with CONNECTION_STRING constant from config.php

File: Model/Product.php [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    {
+        $this->productFileLink = $link;
+    }
+
+    function getProductFileLink()
+    {
+        return $this->productFileLink;
+    }
+
+    // database connection
```
Comment (MohamedEldefrawy) [inline Model/Product.php:37]: remove unnecessary comments

**Review by Muhammed-Ismail2017 — APPROVED** [review]
ok

**Review by MohamedEldefrawy — COMMENTED** [review]
Great work keep it up

**Pull request closed without merging**